app/src/main/java/pl/arturborowiec/kursakademiaandroida/features/episodes/all/presentation/model/EpisodeDisplayable.kt

```diff
@@ -22,5 +23,8 @@ data class EpisodeDisplayable(
         url = episode.url
     )
 
+    @IgnoredOnParcel
+    val fullName = "$code $name"
```

Comment:
W sumie taka ciekawostka jak użyjesz zapisu 
`val fullName
get() = "$code $name"`
to nie będziesz musiał używać tej adnotacji